openshift/templates/issuer-web/config/bcreg-registration/terms-and-conditions.html

```diff
@@ -8,8 +8,5 @@
 </style>
 
 <div class="content">
-  <h3><strong>Terms of Service</strong></h3>
-  <p class="paragraph">
-    This is a proof-of-concept.
-  </p>
-</div>
+	<h2>Terms and Conditions</h2>
+</div>
```

Comment:
This file can be removed altogether as it will be replaced by profile-specific ones at deployment.

Reply:
Okay, wasn't sure if the profile-specific file overrode it so made that file irrelevant, or if it had to be removed. I guess removing it is cleaner.